modules/openapi-generator/src/main/resources/JavaSpring/api.mustache

```diff
@@ -170,7 +173,8 @@ public interface {{classname}} {
             {{#authMethods}}
             @SecurityRequirement(name = "{{name}}"{{#isOAuth}}, scopes={ {{#scopes}}"{{scope}}"{{^-last}}, {{/-last}}{{/scopes}} }{{/isOAuth}}){{^-last}},{{/-last}}
             {{/authMethods}}
-        }{{/hasAuthMethods}}
+        }{{/hasAuthMethods}}{{#hasExternalDocs}},
```

Comment:
Does this also work? I'm not an expert on mustache files, but I think you may not need the `hasExternalDocs` boolean if you check if the parameter `externalDocs` is set like this.

```
}{{/hasAuthMethods}}{{#externalDocs}},
  externalDocs = @ExternalDocumentation(description = "{{externalDocs.description}}", url = "{{externalDocs.url}}")
{{/externalDocs}}
```

Reply:
For example, I see this in the Swift5 templates

```
     {{#externalDocs}}
     - externalDocs: {{.}}
     {{/externalDocs}}
```